app/views/secrets/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="col-sm-12">
+  <div class="text-center">
+    <h2>Create a new secret to send</h2>
+  </div>
+
+  <div class='row'>
+    <div class="col-md-6">
+      <%= simple_form_for @secret, url: secret_url, wrapper: 'horizontal_form', html: {class: 'form-horizontal'} do |f| %>
```

Comment:
There is probably a way to put this whole form (most of the page really) in a partial and DRY it up with the `#new` view